src/utils/incentive.ts

```diff
@@ -185,7 +223,7 @@ export const calculateLeadPercentageRate = (
   // our parachain does not exist yet, no point in calculating incentives
   if (!ownParachain) return BigInt(0);
 
-  // default to 1, to avoit division by 0 errors
+  // default to 1, to avoid division by 0 errors
```

Comment:
This code down looks little scary. Is that all bigInt raw data? Are we sure we are not losing too much precision here?

Reply:
shouldn't we have 0 as default?